website/addons/base/views.py

```diff
@@ -40,15 +40,49 @@
 # import so that associated listener is instantiated and gets emails
 from website.notifications.events.files import FileEvent  # noqa
 
-FILE_GONE_ERROR_MESSAGE = u'''
+ERROR_MESSAGES = {'FILE_GONE': u'''
 <style>
 .file-download{{display: none;}}
 .file-share{{display: none;}}
 .file-delete{{display: none;}}
 </style>
 <div class="alert alert-info" role="alert">
-This link to the file "{file_name}" is no longer valid.
-</div>'''
+<p>
+The file "{file_name}" stored on {provider} was deleted via the OSF.
+</p>
+<p>
+
+It was deleted by <a href="/{deleted_by}">{deleted_by}</a> on {deleted_on}.
```

Comment:
The url and the link text should be different.  This currently tries to build a link the the user's name.  Instead this should create something like `<a href="/mst3k">John Doe</a>`.